NRasterizer.CLI/Program.cs

```diff
@@ -51,8 +52,9 @@ private static void Grayscale(Image image)
 
         private void DrawSoft(FileInfo fontPath, FileInfo target, string text)
         {
+            var lineCount = text.Where(c => c == '\n').Count() + 1;
```

Comment:
yeah, I keep forgetting the overloads on things like count actually take a filter in.